components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/listener/SelfRegistrationCompletionListener.java

```diff
@@ -112,7 +110,8 @@ public boolean isEnabled() {
     @Override
     public boolean doPostExecute(FlowExecutionStep step, FlowExecutionContext flowExecutionContext) {
 
-        if (Constants.COMPLETE.equals(step.getFlowStatus()) && REGISTRATION_FLOW_TYPE.equals(flowExecutionContext.getFlowType())) {
+        if (Constants.COMPLETE.equals(step.getFlowStatus())
```

Comment:
When bump the framework version, the constant has been removed. This change was not related to the intended fix.